jupyterlitexblock/jupyterlitexblock.py

```diff
@@ -126,46 +134,51 @@ def json_response(data):
         return Response(
             json.dumps(data), content_type="application/json", charset="utf8"
         )
-    
+
     def studio_view(self, context=None):
-        notebook_name = os.path.basename(self.default_notebook) if self.default_notebook else ""
+        notebook_name = os.path.basename(
+            self.default_notebook) if self.default_notebook else ""
         studio_context = {
             "jupyterlite_url": self.jupyterlite_url,
             "notebook_name": notebook_name,
-        } 
+        }
         studio_context.update(context or {})
-        template = self.render_template("static/html/upload.html", studio_context)
+        template = self.render_template(
+            "static/html/upload.html", studio_context)
         frag = Fragment(template)
-        frag.add_javascript(self.resource_string("static/js/src/jupyterlitexblock.js"))
+        print("Test test test")
+        frag.add_javascript(self.resource_string(
```

Comment:
why you have span this statement over multiple lines?

Reply:
I have installed wrong formatter auto pep8, I will remove that formatter and update the pr

Reply:
I have installed wrong formatter auto pep8, I will remove that formatter and update the pr